src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -239,14 +239,14 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     }
   `;
 
-  const MostReadSection = styled.section.attrs(() => ({
-    role: 'region',
-    'aria-labelledby': 'Most-Read',
-    'data-e2e': 'most-read',
-  }))``;
+  const MostReadSection = styled.section``;
```

Comment:
Can this not be removed and we simply use the `as` prop when there are no styles to be applied? https://emotion.sh/docs/styled#as-prop